sources/pmwebapi.go

```diff
@@ -110,7 +110,38 @@ func typeLabel(units string, pcpType string, name string) string {
 	} else {
 		name += unit
 	}
+	return name
+}
+
+func fixNaming(name string) string {
 	name = strings.Replace(name, "count / sec", "count_per_second", -1)
+	name = strings.Replace(name, "Kbyte", "kilobyte", -1)
+	name = strings.Replace(name, "nanosec", "nanoseconds", -1)
+	name = strings.Replace(name, "millisec", "milliseconds", -1)
+	name = strings.Replace(name, "failcnt", "failcount", -1)
+	name = strings.Replace(name, "Mbyte", "megabyte", -1)
+	name = strings.Replace(name, "byte", "bytes", -1)
+	name = strings.Replace(name, "sec", "seconds", -1)
+	name = strings.Replace(name, "bytes_byte", "bytes", -1)
+	name = strings.Replace(name, "bytess_bytes", "bytes", -1)
+	name = strings.Replace(name, "megabytesss", "megabytes", -1)
+	name = strings.Replace(name, "kilobytesss", "kilobytes", -1)
+	name = strings.Replace(name, "millisecondsonds", "milliseconds", -1)
+	name = strings.Replace(name, "nanosecondsonds", "nanoseconds", -1)
+	name = strings.Replace(name, "count_per_secondsond", "count_per_second", -1)
+	name = strings.Replace(name, "BLOCK", "block", -1)
```

Comment:
Technically speaking, this and the following lines work... buuuuut you can save yourself a lot of space, make this look cleaner, and catch potentially missed metric names by just using something from the strings package like `ToLower()`.